package.json

```diff
@@ -117,6 +117,13 @@
       "default": true,
       "order": 6
     },
+    "enableLatexmkrc": {
```

Comment:
I'm starting to think we should call this something else. Is it really important to the end user that this functionality is provided via a custom `.latexmkrc` file? That seems like an implementation detail leaking out. We should name this based on the functionality gained by enabling this option.